src/coreclr/jit/CMakeLists.txt

```diff
@@ -626,7 +626,11 @@ endif()
 if (CLR_CMAKE_TARGET_WIN32 AND CLR_CMAKE_PGO_INSTRUMENT)
   # Copy PGO dependency to target dir
   set(PGORT_DLL "pgort140.dll")
-  find_path(PGORT_DIR ${PGORT_DLL} REQUIRED)
+  if(CLR_CMAKE_TARGET_ARCH_ARM64)
+    cmake_path(SET PGORT_DIR "$ENV{VCToolsInstallDir}bin/arm64")
+  else()
```

Comment:
I think you need an extra check here for x86

Reply:
Another option is to keep using `find_path` and leverage `HINTS` options https://cmake.org/cmake/help/latest/command/find_path.html (if the original intent was to disambiguate ${PGORT_DLL} being located)

Reply:
If we use the `HINTS` option then we would have to guarantee that we do not ever have the Arm64 tooling installed on the build machine, or we will always find the Arm64 version, due to the order that `find_path` searchs.

Reply:
I don't know if that is true, but even if it is I think I would prefer to be more explicit. That way we don't get broken by someone changing that invariant.